src/style.css

```diff
@@ -0,0 +1,18 @@
+* {
+  margin: auto;
+}
+
+.red {
+  background-color: red;
+  height: 100px;
+}
+
+.green {
+  background-color: green;
+  height: 100px;
+}
+
+.blue {
+  background-color: blue;
+  height: 100px;
+}
```

Comment:
- all them have similar property, which can be moved to one general class and this class can be added to all the elements that need this style

Reply:
you can add one more class like `block` and add height style to it